src/BinaryParsers/PEBinary/PortableExecutable/PE.cs

```diff
@@ -919,6 +919,14 @@ public bool IsMostlyOptimized(Pdb pdb)
                 Symbol om = omView.Value;
                 ObjectModuleDetails moduleDetails = om.GetObjectModuleDetails();
 
+                // Many non-code-file modules will pass through here.  Referenced libraries will show up as LINK, for example.  Exclude those
+                // from consideration to reduce skew.
+                if ((moduleDetails.Language != Language.C) &&

```

Comment:
This change greatly improved the accuracy of the "is optimized?" check.  At least for a small sample project the majority of modules were libraries and those do not have relevant compiler options.